salviosage/AutoMart
# #165947439 admin dashboard to delete and see whole ads

**What does this PR do?**

- having an admin delete an ads option 

**Description of Task to be completed?**

- admin can go to his dashboard and delete any ad 


**How should this be manually tested?**

- log in and go to admin nav link then click on the ad you want to delete you will see a delete button 


**What are the relevant pivotal tracker stories?**

- #165947439
